modules/ROOT/pages/delete-api-task.adoc

```diff
@@ -1,12 +1,20 @@
-= Removing an API instance on API Manager
+= Remove API Instances
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The procedure removes the capability to manage an API Version from API Manager. If the API resides in Exchange or on the file system as a ZIP, it isn't deleted from Exchange or the file system.
+You can remove a an API instance or a version of an API instance, if you no longer need to manage it from API Manager. when you remove the API version from API Manager, the API is not deleted from Anypoint Exchange (Exchange) or the file system as a ZIP, if it exists in any of these locations.
```

Comment:
* I think the "when" should start with uppercase. 
* Is it ok to name it Anypoint Exchange (Exchange)? sounds redundant
* I would remove the "as a ZIP"

Reply:
I agree that "Anypoint Exchange (Exchange)" sounds redundant. Anypoint Exchange should be enough.

Also, regarding "the file system as a ZIP". I would specify that the API references to an API Asset in Exchange, and such asset won't be deleted.

Reply:
Done! Thanks!